src/lib.rs

```diff
@@ -496,7 +491,7 @@ impl From<Cow<'static, str>> for FastStr {
     }
 }
 
-const INLINE_CAP: usize = 38;
+const INLINE_CAP: usize = 24;
```

Comment:
If you make `Inline` a separate struct and just slap `#[repr(align(8))]` on it you can have `INLINE_CAP = 30` with same performance.

Reply:
Alternatively try reverting all changes and just adding `#[repr(u64)]` to `Repr` enum. This probably will have same effect, as it would change discriminant  type from `u8` to `u64`

Reply:
Hi, thank you very much!
This works! I will change this pr to use this method!

Reply:
Hi, I've tested again on amd64 and found that this method leads to the clone of `Empty` costs 8ns instead of 4ns.
I will revert this use and use the original method.